fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java

```diff
@@ -1348,6 +1348,38 @@ private void initAggregateBuiltins() {
             addBuiltin(
                     AggregateFunction.createBuiltin(GROUP_ARRAY, Lists.newArrayList(t, Type.INT), new ArrayType(t),
                             t, "", "", "", "", "", true, false, true, true));
+
+            //first_value/last_value for array
+            addBuiltin(AggregateFunction.createAnalyticBuiltin("first_value",
+                    Lists.newArrayList(new ArrayType(t)), new ArrayType(t), Type.ARRAY,
+                    "",
+                    "",
+                    null,
+                    "",
+                    ""));
```

Comment:
this register could remove, now all function are vectorized, no need for row batch

Reply:
done!